File: CustomPlayer/TableInfo.vb
Public Class TableInfo
    Public Class Prefix
        Public Name As String
        Public Id As Integer
        Public Value As String
        Public StartTime, EndTime As Date
        Public DurationText As String
        Public Sub New(name As String, id As Integer, value As String, startTime As Date, endTime As Date, durationText As String)
            Me.Name = name
            Me.Id = id
            Me.Value = value
            Me.StartTime = startTime
            Me.EndTime = endTime
            Me.DurationText = durationText
        End Sub
    End Class
    Public Class Suffix
        Inherits Prefix
        Public Sub New(name As String, id As Integer, value As String, startTime As Date, endTime As Date, durationText As String)
            MyBase.New(name, id, value, startTime, endTime, durationText)
        End Sub
    End Class
    Public Class Useing
        Public Name As String
        Public ServerId, PrefixId, SuffixId As Integer
    End Class
    Public Class PlayerList
        Public Name, Commands, Group, ChatColor As String
    End Class
    Public Class GroupList
        Public GroupName, Parent, Commands, ChatColor, Prefix, Suffix As String
    End Class
    Public Class ExpirationInfo
        Public Name, Value, Type As String
        Public StartTime, EndTime As Date
        Public DurationText As String
    End Class
End Class

File: CustomPlayer/TableInfo.cs
namespace CustomPlayer;
#pragma warning disable CS8618 // 在退出构造函数时，不可为 null 的字段必须包含非 null 值。请考虑声明为可以为 null。
public class TableInfo
{
    public class Prefix
    {
        public string Name;
        public int Id;
        public string Value;
        public DateTime StartTime, EndTime;
        public string DurationText;
        public Prefix(string name, int id, string value, DateTime startTime, DateTime endTime, string durationText)
        {
            Name = name;
            Id = id;
            Value = value;
            StartTime = startTime;
            EndTime = endTime;
            DurationText = durationText;
        }
    }
    public class Suffix : Prefix
    {
        public Suffix(string name, int id, string value, DateTime startTime, DateTime endTime, string durationText) : base(name, id, value, startTime, endTime, durationText)
        {
        }
    }
    public class Useing
    {
        public string Name;
        public int ServerId, PrefixId, SuffixId;
    }
    public class PlayerList
    {
        public string Name, Commands, Group, ChatColor;
    }
    public class GroupList
    {
        public string GroupName, Parent, Commands, ChatColor, Prefix, Suffix;
    }
    public class ExpirationInfo
    {
        public string Name, Value, Type;
        public DateTime StartTime, EndTime;
        public string DurationText;
    }
}
#pragma warning restore CS8618 // 在退出构造函数时，不可为 null 的字段必须包含非 null 值。请考虑声明为可以为 null。